evals/calibrate_judge.py

```
@@ -0,0 +1,333 @@
+
+
+def cohen_kappa(human: Sequence[str], machine: Sequence[str]) -> float:
+    """Cohen's kappa for two label sequences (dependency-free).
+
+    1.0 = perfect agreement, 0.0 = chance-level, < 0 = worse than chance.
+    """
+    n = len(human)
+    if n == 0:
+        return 0.0
+    labels = set(human) | set(machine)
+    po = sum(1 for a, b in zip(human, machine) if a == b) / n
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)